ush/setup.sh

```diff
@@ -2481,8 +2484,6 @@ ENKF_FCST="${ENKF_FCST}"
 
 FIX_GSI="${FIX_GSI}"
 FIX_CRTM="${FIX_CRTM}"
-AIRCRAFT_REJECT="${FIX_GSI}"
-SFCOBS_USELIST="${FIX_GSI}"
```

Comment:
This will impact multiple configurations. Were these only used for RTMA, or are they also used for regular DA runs?

Reply:
Yes, it is for all DA-related workflows running on Jet where these two list files should come from the verification group directory where they are updated daily.

Reply:
So should these lines be:
```
AIRCRAFT_REJECT=$AIRCRAFT_REJECT
SFCOBS_USELIST=$SFCOBS_USELIST
```
So that these variables make it into the var_defns.sh file?

Reply:
Yes, I can add these two line to make it more explicitly.